src/types/onyx/ReportActionName.ts

```diff
@@ -0,0 +1,9 @@
+import type CONST from '@src/CONST';
+import type DeepValueOf from '@src/types/utils/DeepValueOf';
+
+/**
+ *
```

Comment:
Let's add a comment or remove the JS doc? If it's required by a newer lint rule, let's add a comment:

```suggestion
 * The name (or type) of a reportAction
```